### DISCUSSION TITLE
force-synthmix ValueError

### DISCUSSION THREAD

**Discussion by Antguz** [davidfrantz/force#120] [Q&A] [answered]
Hi,

I am using `force-synthmix` on Ubuntu 20.04-FORCE 3.7.0. I got the enclosed error regarding npyio.py. Not sure if this is an error on the settings or a bug. 
![Screenshot from 2021-09-03 14-43-39](https://user-images.githubusercontent.com/7254767/132058439-2fb1b1bc-1ad8-4e43-98a4-dee9d57eabc7.png)
 
I have python 3.7.11 and numpy=1.20.3. Enclosed the prm, feature, and class files for a test.
[synthetic.prm.txt](https://github.com/davidfrantz/force/files/7108194/synthetic.prm.txt)
[features.txt](https://github.com/davidfrantz/force/files/7108195/features.txt)
[class.txt](https://github.com/davidfrantz/force/files/7108196/class.txt)

Any idea how to solve this and can you reproduce the error?

Thanks in advance.

**Comment by ernstste** [comment, marked as answer]
Hi,

are you sure your features file (FILE_FEATURES in the parameterization) is ok?
The error points to a missing column. This may be a long shot, but did the whitespace between two values get deleted by any chance?

Stefan

**Comment by Antguz** [comment]
I think that the feature file is ok. There are whitespaces between values. Not sure what the problem is.

Do you have a similar set of files that you can share with me? I would like to test it using files that truly works. 

AG.

**Reply by ernstste** [reply]
The error points to line 3 in your feature file, which is missing a column. Columns are defined by whitespace, this is why I was asking about missing values / whitespace.
I actually missed that you had provided your feature file in the previous post. Line three is indeed missing a value (5 instead of 6 like in lines 1 and 2).

**Reply by Antguz** [reply]
Thanks, you were right.
